README.md

```diff
@@ -32,12 +32,12 @@ Establish an understanding of JSX and a functional component and its parts.
 
 ### **Week 2**
 
-Introduce students to props and the useState Hook.
+Introduce students to props and the `useState` Hook.
 
-**Objective: These two aspects of React are best learned together and are the most vital in creating a React app. We will also go over basic programming concepts in React such as a child component updating its parent's state or its sibling's props.** 
+**Objective: These two aspects of React are best learned together and are the most vital in creating a React app. We will also go over basic programming concepts in React such as a child component updating a parent's state or a sibling's props.** 
 
 - Learn what props are and how they are used to pass information between components
-- Briefly talk about Hooks before going into useState and why state makes React so dynamic
+- Briefly talk about Hooks before going into `useState` and why state makes React so dynamic
 - Go over passing information to a parent or sibling component
 
 **Homework**
```

Comment:
I can't make a comment there (because I didn't edit it), but - what is the Video Player assignment? It's not mentioned anywhere else.

Reply:
The Video Player assignment is the HW for week 2. You can find it in week2/homework/issues